governance/votes/Executive vote - May 10, 2023.md

```diff
@@ -0,0 +1,156 @@
+* **ETH Address A (Coinbase Custody):** TBD
+
+#### Additional Information
+
+Please review the following forum threads containing information about RWA014 (Coinbase Custody) to inform your position before voting:
+* [Initial MIP Forum Thread](https://forum.makerdao.com/t/mip81-coinbase-usdc-institutional-rewards/17703).
+* [CES Domain Team Assessment](https://forum.makerdao.com/t/rwa014-coinbase-custody-interim-ces-domain-team-assessment/19171/1).
+* [Updated Technical Assessment from dewiz](https://forum.makerdao.com/t/rwa014-coinbase-custody-interim-ces-domain-team-assessment/19171/2).
+* [Strategic Finance Core Unit Legal Assessment](https://forum.makerdao.com/t/coinbase-custody-legal-assessment/20384).
+
+### Risk Parameter Changes
+
+As per [this poll](https://vote.makerdao.com/polling/QmYFfRuR) the following parameter changes will be made, if this executive proposal passes:
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Changes
+
+* Increase ETH-A Stability Fee by 0.25% from 1.5% to **1.75%**.
+* Increase ETH-B Stability Fee by 0.25% from 3% to **3.25%**.
+* Increase ETH-C Stability Fee by 0.25% from 0.75% to **1%**.
+* Increase WSTETH-A Stability Fee by 0.25% from 1.5% to **1.75%**.
+* Increase WSTETH-B Stability Fee by 0.25% from 0.75% to **1%**.
+* Increase RETH-A Stability Fee by 0.25% from 0.5% to **0.75%**.
+* Increase CRVV1ETHSTETH-A Stability Fee by 0.25% from 1.5% to **1.75%**.
+
+#### [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) Changes
+
+* Increase the WSTETH-A `gap` by 15 million DAI from 15 million DAI to **30 million DAI**.
+* Increase the WSTETH-B `gap` by 15 million DAI from 15 million DAI to **30 million DAI**.
+* Reduce the WBTC-A `gap` by 10 million DAI from 20 million DAI to **10 million DAI**.
+* Reduce the WBTC-B `gap` by 5 million DAI from 10 million DAI to **5 million DAI**.
+* Reduce the WBTC-C `gap` by 10 million DAI from 20 million DAI to **10 million DAI**.
+
+#### [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) Changes
+
+* Increase the WSTETH-A `ttl` by 21,600 seconds from 21,600 seconds to **43,200 seconds**.
+* Increase the WSTETH-B `ttl` by 28,800 seconds from 28,800 seconds to **57,600 seconds**.
+
+### Starknet Bridge Limit Increase
+
+As per [this initial poll](https://vote.makerdao.com/polling/QmUnhQZy) and this [forum update](https://forum.makerdao.com/t/april-26th-2023-spell-starknet-bridge-limit/20589), the Starknet Bridge Limit will be increased as follows, if this executive proposal passes.
+
+* The Starknet Bridge Limit will be increased by 4 million DAI from 1 million DAI to **5 million DAI**.
+
+### Vesting Stream Setup
+
+As per [MIP106](https://mips.makerdao.com/mips/details/MIP106), the following DAI and MKR vesting streams will be created if this executive proposal passes:
```

Comment:
Worth linking directly to the relevant section? It's quite a long MIP.

Reply:
good suggestion